components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/OperationPoliciesApiServiceImpl.java

```diff
@@ -202,29 +217,65 @@ public Response deleteCommonOperationPolicyByPolicyId(String operationPolicyId,
      * @param messageContext message context
      * @return A list of operation policies available for the API
      */
-    @Override
-    public Response getAllCommonOperationPolicies(Integer limit, Integer offset, String query,
-                                                  MessageContext messageContext) throws APIManagementException {
+    @Override public Response getAllCommonOperationPolicies(Integer limit, Integer offset, String query, String name,
```

Comment:
Why do we need to provide name and version to retrieve all the common policies?

Reply:
since we have a specific endpoint to export the policy, do we need to provide the functionality to getCommonPolicy by name?

Reply:
Fixed by [b7f664a](https://github.com/wso2/carbon-apimgt/pull/11325/commits/b7f664af4e393bb1fe5d307dd9dc9c9107e372d6)